pkg/ocm/clusters.go

```diff
@@ -680,11 +683,21 @@ func (c *Client) createClusterSpec(config Spec, awsClient aws.Client) (*cmv1.Clu
 	}
 
 	if config.Flavour != "" {
-		clusterBuilder = clusterBuilder.Flavour(
-			cmv1.NewFlavour().
-				ID(config.Flavour),
-		)
+		var flavourBuilder *cmv1.FlavourBuilder
+		// If a machine pool root disk is specified, use it to override the default
+		// this only affect the worker nodes
+		if !config.MachinePoolRootDisk.Empty() {
+			flavourBuilder = cmv1.NewFlavour().
```

Comment:
Do we have SRE sign off on this approach, as we are building new flavours, what does the support matrix look like for this? Is there constraints on what disk size a user can set?

Reply:
Discussed offline, handled in the design doc, SRE team is aware of the changes.